### PR TITLE
Query blocks - Support wide alignments in editor.

### PR DESCRIPTION
**Pull request by Addison-Stavlo**
## Description

EDIT - Updated https://github.com/WordPress/gutenberg/pull/24229#issuecomment-665328780

Currently in the site editor, the Query Loop, Query, and Post Content blocks are given a default wrapper that causes their width to be restricted.  A post may have a block with a wide of full block alignment setting.  The post editor, the front end for the post, and the homepage (if set to render current posts) will render the content at full width:
Screen Shot 2020-07-27 at 5 58 02 PM

While in the site editor the width is restricted by the Query Loop and Post Content blocks:
Screen Shot 2020-07-27 at 5 58 26 PM

If we add styles to set the max-width for these blocks inside the editor, or allow for supporting alignment in the Query and Query Loop blocks, they can be less restrictive and in the editor we would have:
Screen Shot 2020-07-27 at 6 06 47 PM

## How has this been tested?

Tested on local docker environment.

* Create a post with some wide/full alignment blocks (such as cover block).
* Load a template that will render this post within the Query or QueryLoop blocks (such as an index template for showing recent posts as homepage).
* Verify that in the editor the wide/full alignment blocks are not restricted in width by the parent query and post content blocks.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by noahtallen** [comment]
I'm not sure technically how, but it sounds like the query loop and post content blocks need to get the same behavior as the normal wrappers for post content on the front end.... I wonder if there are any classnames for that? I also wonder if alignment settings are important in that case -- i.e. it just acts as the "normal" wrapper around the post, if that makes sense

**Comment by Addison-Stavlo** [comment]
That makes sense, however re:
> need to get the same behavior as the normal wrappers for post content

I noticed the Post Content works in a funky way across site editor and front end as well.  That is, with no alignment settings the full width blocks in the Post Content are properly full width on the front end.  If you run this PR to test the index template, you will find adding full width to the query block is not enough.  You must also add full width to the Post Content block in the site editor for its nested blocks to have the full width effect. 🤔

**Comment by noahtallen** [comment]
I see... :/ I think the alignment settings made more sense in a world without nested blocks like this. I don't have any idea what the "correct" behavior would be in this case 🙃

**Comment by Addison-Stavlo** [comment]
OK, I think this (621fb98) makes a bit more sense than adding the align hook.

These blocks don't have actual wrappers on the front end (or in the case of post-content has a 'pseudo-wrapper' with only the "entry-content" class name).  But the editor will treat them as a block and give them a wrapper regardless.  Since these given wrappers in the editor have no specified align settings, they are given the `wp-block` classname which in turn sets their max with to that of responsive content.  This is why these blocks are restricting in the editor and not on the front end.

What I have done is given these blocks a `lightBlockWrapper`, and added `editor.scss` files for them to specifically set their `max-width` property to 100%.  This allows the children blocks that have full/wide alignment to have visual parity inside the editor.  

You may ask "don't we want align settings for post-content?"
It is probably not necessary.  We want post content to be unbiased and unrestrictive towards the blocks it renders.  Alignments can be set by a parent group block (and themes like seedlet blocks seem to already be nesting post content in 'full' alignment set group block).  If we REALLY need to allow alignments for post content, we should consider setting the default value to `full`.  But really, I think allowing the post content to have max-width of 100% and using the group block to control the overall special cases of block alignment makes the most sense.

File: packages/block-library/src/query/edit/index.js [hunk 1]
```diff
@@ -37,7 +41,7 @@ export default function QueryEdit( {
 			<QueryProvider>
 				<InnerBlocks template={ TEMPLATE } />
 			</QueryProvider>
-		</>
+		</Block.div>
```
Comment (Copons) [inline packages/block-library/src/query/edit/index.js:44]: I'm not familiar with this block and I don't see that toolbar, but I wonder if it makes sense to move the `BlockControls` outside the wrapper.

**Review by Copons — APPROVED** [review]
This works great!
I'll pre-approve, in the off-chance my [question](https://github.com/WordPress/gutenberg/pull/24229#discussion_r462305469) doesn't matter.

File: packages/block-library/src/query/edit/index.js (hunk 1 above)
Comment (Addison-Stavlo) [reply packages/block-library/src/query/edit/index.js:44]: Good catch!  It should probably not be wrapping the `BlockControls` as well.

Comment (Addison-Stavlo) [reply packages/block-library/src/query/edit/index.js]: >  I don't see that toolbar

I moved `BlockControls` outside of the wrapper, but I had the toolbar working the same in either case.  Is it better for you now?

**Comment by Addison-Stavlo** [comment]
Updated again - Im leaving the Post-Content block alone for now.  So for testing this you would need to go to the index template and set the post content inside the query blocks to 'full' alignment.  Previously the width would be restricted in the editor but with these changes to the query blocks it should now show full width as expected.

Since it is listed on the FSE block parity issue, Post Content probably should have settings for alignment, colors, etc.  Furthermore, I noticed that here the group block does not restrict the width of its children on the front-end (only in the editor), so my previous assumption about that working as the alignment controller for nested post content does not work at this point in time either.

**Pull request merged**